src/js/other-websites/fill_relay_address.js

```diff
@@ -87,6 +87,11 @@ async function showModal(modalType) {
 
 // eslint-disable-next-line no-redeclare
 function fillInputWithAlias(emailInput, relayAlias) {
+  // BUG: Duplicate fillInputWithAlias calls without proper input content
```

Comment:
I don't quite follow this comment - does this mean that there's still a bug here to be solved, or that this is a workaround for a bug (and if so, how does the workaround work, and how can a future reader verify that the workaround can be removed)? Might be worth expanding the comment.

Reply:
This is a bug to be fixed. Somewhere in the code, this function is getting called with the correct params and it's failing. The one-liner below lets it fail silently, but it should only run once. I'll debug further.

Reply:
:+1: If you can solve it, that's great, but if it works with this workaround, then expanding this comment detailing why the workaround is needed and how it works is good enough for me.

Reply:
Comment expanded in e38a7af9f83c6d6e32b13154d7994ec7529e81d1

Reply:
Bug filed in https://github.com/mozilla/fx-private-relay-add-on/issues/304